analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -105,8 +113,10 @@ public void perform() {
         String groupId = ProjectUtils.getRealGroupId(project);
         String projectName = project.getName();
 
-        if (!configOutput.get().getAndSet(true)) {
-            // Only output the config once to avoid noisy logging.
+        final StringBuilder builder = new StringBuilder(500);
+
+        // Only output the config once to avoid noisy logging.
+        if (logger.isInfoEnabled() && !configOutput.get().getAndSet(true)) {
```

Comment:
Why change this?

Reply:
That was to guard running the code if the logger is not enabled. It is not a simple String, it's a loop for the method called there.

I was having an issue of no test XML file containing the logger output for the alignment task.

Reply:
Re logger : ok.
Re test xml : I don't follow?

Reply:
I don't see any of the output on the console, in files, or anywhere. For example: `grep -r "Starting alignment task for project"` returns no matching logs even though that code is obviously running.